apps/stirling-pdf/0.36.4/docker-compose.yml

```diff
@@ -1,6 +1,6 @@
 services:
   stirling-pdf:
-    image: frooodle/s-pdf:0.36.4
+    image: frooodle/s-pdf:0.36.5
     container_name: ${CONTAINER_NAME}
     restart: always
     networks:
```

Comment:
The provided code difference is a patch file typically used with Git, which shows changes between two versions of the `services` section of your Docker Compose configuration. The main change highlighted is an update to the Docker image version from `0.36.4` to `0.36.5`. This suggests that you may want to verify the compatibility of this new version with other components or services within your infrastructure, particularly if it includes new features or bug fixes. Additionally, ensure that there are no unintended side effects from upgrading the package or ensuring any necessary environment variables remain unchanged.